### PR TITLE
EMUL-8: Homepage: Main Menu

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
## [EMUL-8: Title](https://fourkitchens.atlassian.net/browse/EMUL-8)

**This PR does the following:**
- Adds the main menu to Storybook
- Updates the main menu styles based on the ones in Zeplin

### Local Testing setup (if you haven't done it yet):
- [ ] Clone this repo and/or checkout this branch
- [ ] run `yarn` to install dependencies
- [ ] Copy `.env.example` to `.env.development`, and fill in the required credentials.

### Functional Testing:
- [ ] Run `yarn storybook` and verify the [XXX component](http://localhost:6006/?path=/story/organization-navigation--primary) in storybook.
- [ ] Run `yarn develop` and verify WHATEVER on [the Gatsby site](http://localhost:8000/).

### CODE REVIEW

File: src/components/molecules/cta/cta.module.css [hunk 1]
```diff
@@ -10,7 +10,7 @@
 }
 
 .ctaBg {
-  background-color: #085A7D;
+  background-color: #085a7d;
```
Comment (ryanhagerty) [inline src/components/molecules/cta/cta.module.css:13]: Guessing these are all linter changes and we don't want assign variables yet?

File: src/components/molecules/nav/navigation.tsx [hunk 2]
```diff
@@ -1,51 +1,48 @@
...
 import { Link } from 'gatsby'
 
 import styles from './navigation.module.css'
 
-import Menu from '../../../img/menu.inline.svg'
-import Close from '../../../img/close.inline.svg'
-import Down from '../../../img/down.inline.svg'
+import { Menu } from '../../../img/menu.inline'
+import { Close } from '../../../img/close.inline'
+// import Down from '../../../img/down.inline.svg'
```
Comment (ryanhagerty) [inline src/components/molecules/nav/navigation.tsx]: Do we want to keep this commented out line?

File: src/components/molecules/nav/navigation.tsx [hunk 3]
```diff
@@ -1,51 +1,48 @@
...
-import React, { Component } from 'react'
+import React, { Component, ReactNode } from 'react'
 import { Link } from 'gatsby'
 
 import styles from './navigation.module.css'
 
-import Menu from '../../../img/menu.inline.svg'
-import Close from '../../../img/close.inline.svg'
-import Down from '../../../img/down.inline.svg'
+import { Menu } from '../../../img/menu.inline'
```
Comment (ryanhagerty) [inline src/components/molecules/nav/navigation.tsx:6]: These don't really match the design exactly (Zeplin has chunky straight corners). Is that cool?

File: src/components/molecules/nav/navigation.tsx [hunk 4]
```diff
@@ -1,51 +1,48 @@
...
+
+export default class Nav extends Component<NavProps, State> {
+  readonly state = {
     revealed: false,
     nextShown: false,
   }
 
-  reveal = (e) => {
-    e.preventDefault()
+  reveal: React.MouseEventHandler<HTMLSpanElement> = () => {
```
Comment (ryanhagerty) [inline src/components/molecules/nav/navigation.tsx:28]: Interesting, never seen this before

**Review by ryanhagerty — COMMENTED** [review]
Couple minor questions than it's GTG for me

File: src/components/molecules/cta/cta.module.css (hunk 1 above)
Comment (ModulesUnraveled) [reply src/components/molecules/cta/cta.module.css:13]: Correct on both accounts.

File: src/components/molecules/nav/navigation.tsx (hunk 2 above)
Comment (ModulesUnraveled) [reply src/components/molecules/nav/navigation.tsx]: No, I'll go ahead and delete it. There was apparently some work-in-progress to support multi-level menu, but for now, at least, it's just going to be a single level, so I ripped out the other parts. I'll remove this too

File: src/components/molecules/nav/navigation.tsx (hunk 3 above)
Comment (ModulesUnraveled) [reply src/components/molecules/nav/navigation.tsx:6]: Yeah, for now, I'm just going to use the existing svgs. We can update later, if we decide that's necessary

File: src/components/molecules/nav/navigation.tsx (hunk 4 above)
Comment (ModulesUnraveled) [reply src/components/molecules/nav/navigation.tsx:28]: Me either, but I needed to do it in order to make typescript happy :)

File: src/components/molecules/cta/cta.module.css (hunk 1 above)
Comment (ModulesUnraveled) [reply src/components/molecules/cta/cta.module.css:13]: I'm working on defining/using variables as I go through and update/create the components. So all of these changes are just from the linter auto-fixing stuff.

File: src/components/molecules/nav/navigation.tsx (hunk 2 above)
Comment (ModulesUnraveled) [reply src/components/molecules/nav/navigation.tsx]: Done

**Pull request merged**